pyproject.toml

```
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "zcash-test-vectors"
+version = "0.1.0"
+description = "Zcash test vectors"
+authors = [
+]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/zcash-hackworks/zcash-test-vectors/"
+repository = "https://github.com/zcash-hackworks/zcash-test-vectors/"
+documentation = "https://github.com/zcash-hackworks/zcash-test-vectors/"
```

Per https://github.com/python-poetry/poetry/issues/1537#issuecomment-854612263:
```suggestion
documentation = "https://github.com/zcash-hackworks/zcash-test-vectors/"
classifiers = [
    "Private :: Do Not Upload",
]
```